packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -572,6 +574,7 @@ export default class TradeStore extends BaseStore {
                             this.debouncedProposal();
                             this.clearLimitOrderBarriers();
                             this.pushPurchaseDataToGtm(contract_data);
+                            this.throttleDisableIsPurchasingContract();
```

Comment:
why do we need to throttle? I think this will work even without throttling.

Reply:
@balakrishna-binary I tried this with @reza-fs and see this issue didn't resolve the issue. because mainly issue happens when user clicks on the purchase repeatedly